### ISSUE TITLE
Unable to re-use previously used environment settings in Invoke-Build

### ISSUE THREAD

**Issue by virmitio** [coapp/coapp.powershell#4]
v 1.23.187

Attempting to use `.SetEnvironmentFromTarget` on a previously used target environment results in the environment being declared as invalid.  It appears that the actual environment variables are replaced appropriately, but the variable which declares it as valid is set improperly.

eg.

``` c#
.SetEnvironmentFromTarget.Target = "VC_V110_X64";
.Message.Text = "$(IsEnvironmentValid)";  // true
.SetEnvironmentFromTarget.Target = "VC_V110_X64";
.Message.Text = "$(IsEnvironmentValid)";  // false
```

**Comment by fearthecowboy** [comment]
Fixed in  1.23.196.0 -- use :

``` powershell
# Update CoApp PowerShell Tools
Update-CoAppTools -KillPowerShells -Beta 
```

**Comment by virmitio** [comment]
Confirmed fixed in 1.23.196
